### ISSUE TITLE
Master Infusion crystal consumed in auto crafting

### ISSUE THREAD

**Issue by Gumbofizz** [Cazadorsniper/PO3#1631]
3.4.6
Pack Mode :Normal
Singleplayer

Master Infusion crystal consumed when auto crafting with it.
From what I can see not much was previously reported, my mistake if so.

Set recipe for essence auto crafting and when crafting it consumes it

**Comment by Cazadorsniper** [comment]
automating with AE? Try doing it with RFTools.

**Comment by Gumbofizz** [comment]
Yes auto crafting with AE2. Recipe set to use master infusion crystal . I tried it eith oredix on and off, neither of them working. The calculation picks up that the crystal is in the system and start crafting but it uses the crystal and it never returns to the system then the crafting wants to craft more crystals beacuse of the fact that the crystal was used. 

I tried using the crater from RFtools but I am struggling to get the system to always keep 64 of each essence available after converting each essence to the next tier for more compact storage and to teach the ME system to rather break down the essences as needed.
